From 1407561faee5a8d2a7fe670d2dba6b3c2c7d8071 Mon Sep 17 00:00:00 2001
From: liuW3X <weix.w.liu@intel.com>
Date: Tue, 20 Mar 2018 15:10:01 +0800
Subject: [PATCH 0694/1240] linux/msi.h: compiling fails with arm/arm64 arch

__pci_write_msi_msg has its' own implemention on x86 arch,
using CONFIG_X86 to distinguish x86 arch from non-x86 arch

Change-Id: Iaf1b472c2cb982f732f6a88c6883c3932e2af82d
Signed-off-by: liuW3X <weix.w.liu@intel.com>
---
 include/linux/msi.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/msi.h b/include/linux/msi.h
index db1f5f8..286fa2e 100644
--- a/include/linux/msi.h
+++ b/include/linux/msi.h
@@ -141,7 +141,7 @@ void free_msi_entry(struct msi_desc *entry);
 void __pci_read_msi_msg(struct msi_desc *entry, struct msi_msg *msg);
 
 void native_write_msi_msg(struct msi_desc *entry, struct msi_msg *msg);
-#ifdef CONFIG_PARAVIRT
+#if defined(CONFIG_PARAVIRT) && defined(CONFIG_X86)
 #include <asm/paravirt.h>
 #define __pci_write_msi_msg write_msi_msg_paravirt
 #else
-- 
2.7.4

